lib/functions-render.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+// RENDERS
+
+function render_embed_caption($caption) {
+  echo '<h5 class="text-align-center margin-top-tiny margin-bottom-tiny">' . $caption . '</h5>';
+}
+
+function render_divider() {
+?>
+<div class="grid-item item-s-12">
+  <div class="dotted-divider-full"></div>
+</div>
+<?php
+}
+
+function render_divider_with_content($content) {
+  if (!$content) {
+    render_divider();
+  }
+?>
```

Comment:
could these be one function? `render_divider($content)` with a condition for content?